### PR TITLE
convert-examples-repo-to-defs cloud nux quickstart_snowflake

### PR DESCRIPTION
**Pull request by yuhan**
### Summary & Motivation
convert quickstart_snowflake example after it's moved from a standalone repo back to the mono repo (https://github.com/dagster-io/dagster/pull/11132)

- `workspace.yaml` -> `pyproject.toml`
- `@repository` -> `Definitions`
- `repository.py` -> `__init__.py`

### How I Tested These Changes
- local dagit loads
- unit

### CODE REVIEW

**Comment by yuhan** [comment]
Current dependencies on/for this PR:
* master
  * **PR #11122** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11122" target="_blank">Graphite</a> 
    * **PR #11130** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11130" target="_blank">Graphite</a> 
      * **PR #11131** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11131" target="_blank">Graphite</a> 
        * **PR #11132** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11132" target="_blank">Graphite</a> 
          * **PR #11135** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11135" target="_blank">Graphite</a> 
            * **PR #11136** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11136" target="_blank">Graphite</a> 
              * **PR #11137** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11137" target="_blank">Graphite</a>  👈
    * **PR #11126** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/11126" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/11137?utm_source=stack-comment).

File: examples/quickstart_snowflake/quickstart_snowflake/__init__.py [hunk 1]
```diff
@@ -1 +1,35 @@
...
+
+daily_refresh_schedule = ScheduleDefinition(
+    job=define_asset_job(name="all_assets_job"), cron_schedule="0 0 * * *"
+)
+
+
+defs = Definitions(
+    assets=load_assets_from_package_module(assets),
+    resources={
+        "io_manager": build_snowflake_io_manager([SnowflakePandasTypeHandler()]).configured(
```
Comment (schrockn) [inline examples/quickstart_snowflake/quickstart_snowflake/__init__.py:21]: switch to `snowflake_pandas_io_manager`

**Review by schrockn — DISMISSED** [review]

File: examples/quickstart_snowflake/quickstart_snowflake/__init__.py (hunk 1 above)
Comment (yuhan) [reply examples/quickstart_snowflake/quickstart_snowflake/__init__.py:21]: good catch! this was built right before that was released so i totally forgot to switch

Comment (yuhan) [reply examples/quickstart_snowflake/quickstart_snowflake/__init__.py]: updated

**Review by schrockn — APPROVED** [review]

**Comment by yuhan** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/11137?utm_source=stack-comment).


Job Reference: gY2QLTjNrPJtUd2KNqDS

**Pull request merged**